api/MediaDevices.json

```diff
@@ -230,28 +230,14 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaDevices/getDisplayMedia",
           "description": "<code>getDisplayMedia()</code>",
           "support": {
-            "chrome": [
-              {
-                "version_added": "72"
-              },
-              {
-                "version_added": "70",
-                "version_removed": "72",
-                "alternative_name": "navigator.getDisplayMedia",
-                "notes": "Available as a member of <code>Navigator</code> instead of <code>MediaDevices</code> in Chrome 70 and 71."
-              }
-            ],
-            "chrome_android": [
-              {
-                "version_added": "72"
-              },
-              {
-                "version_added": "70",
-                "version_removed": "72",
-                "alternative_name": "navigator.getDisplayMedia",
-                "notes": "Available as a member of <code>Navigator</code> instead of <code>MediaDevices</code> in Chrome 70 and 71."
-              }
-            ],
+            "chrome": {
+              "version_added": "70",
+              "notes": "Available as a member of <code>Navigator</code> instead of <code>MediaDevices</code> in Chrome 70 and 71."
+            },
+            "chrome_android": {
+              "version_added": "70",
+              "notes": "Available as a member of <code>Navigator</code> instead of <code>MediaDevices</code> in Chrome 70 and 71."
+            },
```

Comment:
Is this really the way this should be handled? I would have thought the old way would be better.